protocol-units/da/m1/light-node/src/v1/passthrough.rs

```diff
@@ -146,7 +146,7 @@ impl LightNodeV1 {
 		Ok(verified_blobs)
 	}
 
-	#[tracing::instrument(target = "movement_timing", level = "debug")]
+	#[tracing::instrument(target = "movement_telemetry", level = "debug")]
```

Comment:
I would like to have comments on each of these explaining why they are needed to achieve a certain metric as outlined here: 
- https://github.com/movementlabsxyz/movement/discussions/367
- https://github.com/movementlabsxyz/sre/discussions/63

Reply:
I'm not sure if we need all of these for telemetry. Originally they were added for perf research.
Do you think we can just remove those that don't contribute to a particular metric goal, or convert them to a plain spans/events with the default target?